airbyte-integrations/bases/base-normalization/entrypoint.sh

```diff
@@ -41,17 +77,11 @@ function main() {
 
   case "$CMD" in
   run)
-    cp -r /airbyte/normalization_code/dbt-template/* $PROJECT_DIR
-    transform-config --config "$CONFIG_FILE" --integration-type "$INTEGRATION_TYPE" --out $PROJECT_DIR
-    transform-catalog --integration-type "$INTEGRATION_TYPE" --profile-config-dir $PROJECT_DIR --catalog "$CATALOG_FILE" --out $PROJECT_DIR/models/generated/ --json-column _airbyte_data
-    dbt run --profiles-dir $PROJECT_DIR --project-dir $PROJECT_DIR
+    configuredbt
+    dbt run --profiles-dir "${PROJECT_DIR}" --project-dir "${PROJECT_DIR}"
     ;;
-  dry-run)
```

Comment:
nit: is having a dry-run option still useful?

I don't often run the dbt workflow separately so I'm just wondering.

Reply:
no, the dry-run option was never used anywhere